cwms_radar_api/build.gradle

```diff
@@ -12,14 +12,19 @@ dependencies {
         exclude group: "com.oracle", module: "*"
     }
     implementation "com.googlecode.owasp-java-html-sanitizer:owasp-java-html-sanitizer:$OWASP_SANITIZER_VERSION"
-    
-    compileOnly "io.javalin:javalin:$JAVALIN_VERSION"
+
+    implementation "io.javalin:javalin:$JAVALIN_VERSION"
```

Comment:
This may need to say compileOnly here and be added to the _standalone build.gradle. There's a bunch of exclusions required since Javalin by default embeds Jetty. But we can worry about it later.